docs/installation/mattermost/index.md

```diff
@@ -58,27 +58,6 @@ Export the channel name as an environment variable:
 export MATTERMOST_CHANNEL="{channel_name}"
```

Comment:
Should we also update

```
BotKube is a messaging bot for monitoring and debugging Kubernetes clusters. Visit https://botkube.io/usage for help..
```
for Discord? Instead of pointing to the `/usage`, should we update it with "Run `@BotKube help` command for usage"?

Reply:
For MS Teams we have:
```
App that helps you monitor your Kubernetes cluster, debug critical deployments & gives recommendations for standard practices.
```
so maybe it would be good to unify that but don't put in app description `@BotKube help`?

So use the one from our README?
```
BotKube helps you monitor your Kubernetes cluster, debug critical deployments and gives recommendations for standard practices by running checks on the Kubernetes resources.
```

Reply:
![tenor-255083166](https://user-images.githubusercontent.com/7155799/191012500-d77b6538-533c-4acc-8594-b60cb2376f5a.gif)